src/lisflood/global_modules/add1.py

```diff
@@ -907,15 +907,15 @@ def writenet(flag, inputmap, netfile, DtDay,
             nf1.variables["time"][:] = date2num(time_stamps, time.units, time.calendar)
             # for i in metadataNCDF['time']: exec('%s="%s"') % ("time."+i, metadataNCDF['time'][i])
             if 'x' in meta_netcdf.data:
-                value = nf1.createVariable(prefix, data_format, ('time', 'y', 'x'), zlib=True, fill_value=-9999, chunksizes=(1, row, col))
+                value = nf1.createVariable(prefix, 'd', ('time', 'y', 'x'), zlib=True, fill_value=-9999, chunksizes=(1, row, col))
```

Comment:
Now we write all variables with 'd' format so data_format parameter in writenet is not used. We need to remove it from function signature and from all writenet function calls.

Reply:
When writing with 'd' format files become larger. Some users might not be interested in warm start but might prefer to have smaller files. data_format could become an option (with default equal to 'd'). What do you think?

Reply:
Do you mean:

1) Remove the parameter from function signature and calls
2) Adding an option with default to "d" so that user can overwrite it in XML settings. The option is only read and used inside writenet function.

Cool?

I agree. If you can proceed in this way is perfect.

Reply:
Hallo Domenico,
it seems I've got a few problems adding a new option. Code is quite different now... I would suggest to merge this version fist, so Ervin can go on with testing Glofas and we have a version that can correctly warm start. Then we take care of the option for writing in single precision. Could we do this?
Could we do this?

Reply:
Hi Cinzia,
yes this will work. 
You can go ahead then I can take care of adding that option, also. No problem.